src/pages/Thanks/ThanksPage.vue

```diff
@@ -289,6 +300,21 @@ export default {
 				);
 			}
 		}
+
+		// CORE-427 Thanks auto deposit upsell experiment
+		const autoDepositUpsellExp = this.apollo.readFragment({
```

Comment:
To avoid confusion, let's rename this variable to something like `simpleSocialShareExp`

Reply:
done!